LanguageFeatures/Parts-with-imports/scope_A03_t03_part1.dart

```diff
@@ -19,7 +19,7 @@
 
 part of 'scope_A03_t03.dart';
 
-import 'scope_lib1.dart' deferred as d;
+import 'scope_lib1.dart' deferred as d hide LibExt;
```

Comment:
Just wondering, why would `LibExt` be subject to a `hide` here?

Reply:
Otherwise `deferred_import_of_extension` error occurs. Hiding of this extension eleminates the error.

Reply:
Oh! (checking specs...)

Reply:
OK. "This is a temporary restriction ensuring that no extensions are introduced using deferred imports, allowing us to later introduce semantics for such extensions without affecting existing code". It was too scary. :grinning:

Reply:
We have no such rule for extension types.

Reply:
(And I'm not sure why an extension or an extension type should be more scary than a top-level function. After all, that's how they are compiled anyway.)